### PR TITLE
WIP security/maltrail: new plugin

### PR DESCRIPTION
**Pull request by mimugmail**
Hi,

new plugin, consists of 2 services, server and sensor. You can run on one system both, or just central server or a sensor sending logs to a server. 

Form/Model seems to be ok, but I get a configd error and therefore config file doesn't get touched:
```
Mar 11 09:38:05 OPNsense configd.py: [9891a9de-bbca-4718-9c47-a166eb73943b] stopping Maltrail Server
Mar 11 09:38:05 OPNsense configd.py: [ef529808-2bce-45f0-a1e2-fef2f283aa47] generate template OPNsense/Maltrail
Mar 11 09:38:05 OPNsense configd.py: generate template container OPNsense/Maltrail
Mar 11 09:38:05 OPNsense configd.py: [ef529808-2bce-45f0-a1e2-fef2f283aa47] Inline action failed with OPNsense/Maltrail/maltrailserver at Traceback (most recent call last):   File "/usr/local/opnsense/service/modules/processhandler.py", line 509, in execute     return ph_inline_actions.execute(self, inline_act_parameters)   File "/usr/local/opnsense/service/modules/ph_inline_actions.py", line 51, in execute     filenames = tmpl.generate(parameters)   File "/usr/local/opnsense/service/modules/template.py", line 332, in generate     raise render_exception TemplateNotFound: OPNsense/Maltrail/maltrailserver
Mar 11 09:38:05 OPNsense configd.py: [47311867-5075-49e3-8369-5869137a53e0] request Maltrail Server status
Mar 11 09:38:05 OPNsense configd.py: [80c9ae6e-6bfa-4068-b1a9-bed91d423cdf] starting Maltrail Server

```

Server start/stop also works see for me it seems it should work in general, but I have no idea why this comes:
`TemplateNotFound: OPNsense/Maltrail/maltrailserver`

Please don't review other stuff, it's not yet finished

### CODE REVIEW

File: security/maltrail/src/opnsense/service/templates/OPNsense/Maltrail/+TARGETS [hunk 1]
```diff
@@ -0,0 +1,3 @@
+maltrailsensor:/etc/rc.conf.d/opnsense-maltrailsensor
```
Comment (fichtner) [inline security/maltrail/src/opnsense/service/templates/OPNsense/Maltrail/+TARGETS]: here's your error:

```suggestion
maltrailsensor.conf:/etc/rc.conf.d/opnsense-maltrailsensor
```

File: security/maltrail/src/opnsense/service/templates/OPNsense/Maltrail/+TARGETS [hunk 2]
```diff
@@ -0,0 +1,3 @@
...
+maltrailserver:/etc/rc.conf.d/opnsense-maltrailserver
```
Comment (fichtner) [inline security/maltrail/src/opnsense/service/templates/OPNsense/Maltrail/+TARGETS]: and here:

```suggestion
maltrailserver.conf:/etc/rc.conf.d/opnsense-maltrailserver
```

**Comment by fichtner** [comment]
although it would be better to rename the rc.conf files to avoid the ".conf" suffix, it's a bit misleading

**Comment by mimugmail** [comment]
I always thought I brought my c+p skills to perfection :) You made my day, thx 👍

**Pull request closed without merging**